src/js/components/Pagination/Pagination.js

```diff
@@ -176,9 +178,41 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    const itemsBegin = step * (activePage - 1) + 1;
+    const itemsEnd = itemsBegin + step - 1;
+    let showSummary;
+    if (showSummaryProp === true)
+      showSummary = (
+        <Text>
+          Showing {itemsBegin} - {itemsEnd} of {numberItems}
+        </Text>
+      );
+    else showSummary = showSummaryProp;
+
+    // only apply these if the caller hasn't already specified their own theming
+    // otherwise, let the caller control
+    const showSummaryStyles =
+      showSummary && !theme.pagination.container
+        ? {
+            align: 'center',
+            fill: 'horizontal',
+            direction: 'row',
+            justify: 'between',
+          }
+        : undefined;
+
     return (
-      <StyledPaginationContainer {...theme.pagination.container} {...rest}>
-        <Nav a11yTitle={a11yTitle || 'Pagination Navigation'} ref={ref}>
+      <StyledPaginationContainer
+        {...theme.pagination.container}
+        {...showSummaryStyles}
+        {...rest}
+      >
+        {showSummary}
+        <Nav
+          a11yTitle={a11yTitle || 'Pagination Navigation'}
+          ref={ref}
+          {...theme.pagination.nav}
```

Comment:
I think the `.nav` naming wouldn't immediately be clear to me. Even though semantically this is a nav, as an implementor my mind doesn't immediately go here.

I think about the component being:
```
<Container>
    <Summary />
    <Controls>
        <Control0 />
        <Control1 />
        ...
        <ControlN /> 
    </Controls>
</Container>
```

Reply:
Very true, ideally we wouldn't have to add this at all but see comment above on use case demonstrated. if there's a way to get around it, we can remove this. If we need to keep it, we can think of more clear naming.